data-shell/north-pacific-gyre/2012-07-03/goostats

```diff
@@ -1,11 +1,10 @@
-while getopts J:r: option
-do
-    case "$option" in
-    J) Jarg="$OPTARG"
-       shift $((OPTIND));;
-    r) shift $((OPTIND-1));;
-    esac
-done
+# check for the right number of input arguments
+if [ $# -ne 2 ]
+  then
+    echo "goodstat file1 file2"
```

Comment:
Small typo: `goodstat` -> `goostats`

Reply:
@gcapes 
How about this justification of the script: 
Nelle's supervisor insisted that all her analytics must be reproducible. The easiest way to capture all the steps is in a script.